e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -64,6 +65,9 @@
 
     private static final String FHIR_TYPE = "application/fhir+ndjson";
 
+    private static final String CURRENCY_IDENTIFIER =
```

Comment:
This is a duplicate definition.  Please consolidate into using only one.

Reply:
Can't dedup this one because it isn't a shared dependency of the worker and I don't want to move it to the common module.